### DISCUSSION TITLE
Playback of track using multiple channels (surround/Dolby Atmos)

### DISCUSSION THREAD

**Discussion by Reached** [katspaugh/wavesurfer.js#2680] [Q&A]
Hi there,

We are using Wavesurfer with great success (love it!).

One of our users are trying to playback a wav file containing multiple channels on a mix with Dolby Atmos, however it errors out when attempting to render the waveform. 

Has anyone had any success playing back files like this, or is it even possible? :)

Best regards, Casper

**Comment by thijstriemstra** [comment]
thanks for testing this.

Please include your wavesurfer.js config, and the output of `ffprobe` (part of ffmpeg) for the audio file.

> However it errors out when attempting to render the waveform.

What errors specifically?
